Tests/SentryTests/SentryScopeSwiftTests.swift

```diff
@@ -214,4 +214,70 @@ class SentryScopeSwiftTests: XCTestCase {
         
         XCTAssertEqual(1, attachments.count)
     }
+    
+    // Altough we only run this test above the below specified versions, we exped the
+    // implementation to be thread safe
+    @available(tvOS 10.0, *)
+    @available(OSX 10.12, *)
+    @available(iOS 10.0, *)
+    func testModifyingScopeFromMultipleThreads() {
+        let queue = DispatchQueue(label: "SentryScopeTests", qos: .utility, attributes: [.concurrent, .initiallyInactive])
```

Comment:
Try to get these threads a higher priority:
```suggestion
        let queue = DispatchQueue(label: "SentryScopeTests", qos: .userInteractive, attributes: [.concurrent, .initiallyInactive])
```